frontend/public/components/dashboard/project-dashboard/activity-card.tsx

```diff
@@ -141,32 +141,57 @@ const OngoingActivity = connect(mapStateToProps)(
   ),
 );
 
+const RecentEventFooter = withDashboardResources(
+  ({
+    watchK8sResource,
+    stopWatchK8sResource,
+    resources,
+    projectName,
+    viewEvents,
+  }: DashboardItemProps & { projectName: string; viewEvents: string }) => {
+    const { t } = useTranslation();
+    React.useEffect(() => {
+      if (projectName) {
+        const eventsResource = getEventsResource(projectName);
+        watchK8sResource(eventsResource);
+        return () => {
+          stopWatchK8sResource(eventsResource);
+        };
+      }
+    }, [watchK8sResource, stopWatchK8sResource, projectName]);
+
+    const events = resources.events as FirehoseResult<EventKind[]>;
+    const shouldShowFooter = events?.loaded && events?.data && events.data.length > 50;
+
+    if (!shouldShowFooter) {
+      return null;
+    }
+
+    return (
+      <CardFooter>
+        <Link to={viewEvents} data-test="events-view-all-link">
+          {t('console-shared~View all events')}
+        </Link>
+      </CardFooter>
+    );
```

Comment:
should add the divider again

```suggestion
    return (
      <>
        <Divider />
        <CardFooter>
          <Link to={viewEvents} data-test="events-view-all-link">
            {t('console-shared~View all events')}
          </Link>
        </CardFooter>
      </>
    );
```